cni/pkg/nodeagent/net.go

```diff
@@ -125,9 +126,22 @@ func (s *NetServer) AddPodToMesh(ctx context.Context, pod *corev1.Pod, podIPs []
 		return err
 	}
 
+	// If true, the pod will run in 'ingress mode'. This is intended to be used for "ingress" type workloads which handle
+	// non-mesh traffic on inbound, and send to the mesh on outbound.
+	// Basically, this just disables inbound redirection.
+	// We use the SidecarTrafficExcludeInboundPorts annotation for compatibility (its somewhat widely used) but don't support all values.
+	ingressMode := false
+	if a, f := pod.Annotations[annotation.SidecarTrafficExcludeInboundPorts.Name]; f {
```

Comment:
It's more confusing to reuse this here IMO where it works for some but not all old-style values, and it's not really intended for this purpose, we should just add a label:

`ambient.istio.io/disableInboundCapture: true`

or maybe

`gateway.istio.io/disableInboundCapture: true` / `gateway.istio.io/captureOutboundOnly: true`

Reply:
:+1: done